test/e2e/storage/testsuites/topology.go

```diff
@@ -329,6 +329,9 @@ func (t *topologyTestSuite) createResources(ctx context.Context, cs clientset.In
 		SeLinuxLabel:  e2epod.GetLinuxLabel(),
 		ImageID:       e2epod.GetDefaultTestImageID(),
 	}
+	if framework.NodeOSDistroIs("windows") {
```

Comment:
`l.config.ClientNodeSelection` should also be available here, any reason why we can't use that in the assignment?

Reply:
Done.